src/Terminus.php

```diff
@@ -363,7 +364,9 @@ private function addBuiltInCommandsAndHooks()
             'Pantheon\\Terminus\\Commands\\NewRelic\\DisableCommand',
             'Pantheon\\Terminus\\Commands\\NewRelic\\EnableCommand',
             'Pantheon\\Terminus\\Commands\\NewRelic\\InfoCommand',
+            'Pantheon\\Terminus\\Commands\\Org\\InfoCommand',
             'Pantheon\\Terminus\\Commands\\Org\\ListCommand',
+            'Pantheon\\Terminus\\Commands\\Org\\OrgCommand',
```

Comment:
Do we really need to add a base class here?

Reply:
Removed it.